src/test/java/com/aws/greengrass/tes/TokenExchangeServiceTest.java

```diff
@@ -111,7 +111,7 @@ void setup() {
                 .thenReturn(componentTypeTopic);
         when(configuration.lookup(SERVICES_NAMESPACE_TOPIC, DEFAULT_NUCLEUS_COMPONENT_NAME, CONFIGURATION_CONFIG_KEY,
                 COMPONENT_STORE_MAX_SIZE_BYTES)).thenReturn(componentStoreSizeLimitTopic);
-        when(configuration.lookupTopics(SERVICES_NAMESPACE_TOPIC, DEFAULT_NUCLEUS_COMPONENT_NAME, CONFIGURATION_CONFIG_KEY,
+        lenient().when(configuration.lookupTopics(SERVICES_NAMESPACE_TOPIC, DEFAULT_NUCLEUS_COMPONENT_NAME, CONFIGURATION_CONFIG_KEY,
```

Comment:
out of curiosity, why lenient?

Reply:
you're right, I can remove this line. I initially  made this lenient before moving the `handleLoggingConfig()` call to Kernel